app/models/notification_setting.rb

```diff
@@ -3,15 +3,37 @@
 #
 # Table name: notification_settings
 #
-#  id           :integer          not null, primary key
-#  setting_name :string           not null
-#  created_at   :datetime         not null
-#  updated_at   :datetime         not null
+#  id                      :integer          not null, primary key
+#  is_email_toggled        :boolean          default(TRUE)
+#  is_fb_messenger_toggled :boolean          default(TRUE)
+#  is_mobile_toggled       :boolean          default(TRUE)
+#  is_web_toggled          :boolean          default(TRUE)
```

Comment:
prefixing these predicates with `is` is out-of-place in Ruby, generally we'd do `web_enabled`, `mobile_enabled`, etc. — and then Rails automatically gives us `web_enabled?` methods on the model